src/Command/UpdateConferencesCoordinatesCommand.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Command;
+
+use App\Fetcher\LocationGuesser;
+use App\Repository\ConferenceRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class UpdateConferencesCoordinatesCommand extends Command
+{
+    private ConferenceRepository $conferenceRepository;
+    private LocationGuesser $locationGuesser;
+    private EntityManagerInterface $manager;
+
+    public function __construct(ConferenceRepository $conferenceRepository, LocationGuesser $locationGuesser, EntityManagerInterface $manager)
+    {
+        $this->conferenceRepository = $conferenceRepository;
+        $this->locationGuesser = $locationGuesser;
+        $this->manager = $manager;
+
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this->setName('starfleet:conferences:update-coordinates');
+        $this->setDescription('Add coordinates to conferences already stored in the database.');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $allConferences = $this->conferenceRepository->findAll();
```

Comment:
I would be better to use an iterator to no consume too much memory `$qb->getQuery()->iterate()`
Then, you'll need to flush + clear the EM in the loop